rotkehlchen/utils/validation_schemas.py

```diff
@@ -0,0 +1,24 @@
+from marshmallow import Schema, fields
+
+from rotkehlchen.api.v1.encoding import UnderlyingTokenInfoSchema
+
+
+class AssetListSchema(Schema):
+    identifier = fields.String(required=True)
+    name = fields.String(required=True)
+    symbol = fields.String(required=True)
+    asset_type = fields.String(required=True)
+    forked = fields.String(required=True, allow_none=True)
```

Comment:
Why `required = True` and `allow_none=True` and not `required=False` and `load_default=None`?

Reply:
In the json file I export everything so I felt that if the user is importing the rotki exported file it should have the same format. I believe there is no performance hit or similar